src/compiler/emitter.ts

```diff
@@ -2505,6 +2507,87 @@ var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, Promi
                 }
             }
 
+            /**
+             * Emit ES7 exponentiation operator downlevel using Math.pow
+             * @param node {BinaryExpression} a binary expression node containing exponentiationOperator (**, **=) 
+             */
+            function emitExponentiationOperator(node: BinaryExpression) {
+                let leftHandSideExpression = node.left;
+                if (node.operatorToken.kind === SyntaxKind.AsteriskAsteriskEqualsToken) {
+                    let synthesizedLHS: ElementAccessExpression | PropertyAccessExpression;
+
+                    // This is used to decide whether to emit parenthesis around the expresison.
+                    // Parenthesis is required for following cases:
+                    //      capture variable while emitting right-hand side operand.
+                    //          a[0] **= a[0] **= 2
+                    //          _a = a, _a[0] = Math.pow(_a[0], (_b = a, _b[0] = Math.pow(_b[0], 2)));
+                    //                                          ^ -> required extra parenthesis controlled by shouldEmitParenthesis
+                    //      exponentiation compount in variable declaration
+                    //          var x = a[0] **= a[0] **= 2
+                    //          var x = (_a = a, _a[0] = Math.pow(_a[0], (_b = a, _b[0] = Math.pow(_b[0], 2))));
+                    //                  ^ -> required extra parenthesis controlled by shouldEmitParenthesis
+                    // Otherwise, false
+                    let shouldEmitParenthesis = false;
+
+                    if (isElementAccessExpression(leftHandSideExpression)) {
+                        shouldEmitParenthesis = node.parent.kind === SyntaxKind.VariableDeclaration || node.parent.kind === SyntaxKind.BinaryExpression;

```

Comment:
as we discussed offline, i will remove this logic and always emit with parenthesis as you already have a PR to fix this issue with parenthesis